src/app/layout/footer/footer.component.ts

```diff
@@ -7,6 +8,11 @@ import { Component, OnInit } from '@angular/core';
 })
 export class FooterComponent implements OnInit {
   currentYear: number;
+  nameLinks = [
+    new NetworkLink("Facebook", "https://es-la.facebook.com/"),
+    new NetworkLink("Twiter", "https://twitter.com/"),
+    new NetworkLink("Instagram", "https://www.instagram.com/")
```

Comment:
Use single quotes in TypeScript, double quotes in HTML. In this case, this would be:
```
new NetworkLink('Instagram', 'https://www.instagram.com/')
```